docs/blog/newsletter-july-2024.rst

```diff
@@ -0,0 +1,136 @@
+:og:image: https://www.writethedocs.org/_static/logo-opengraph.png
+
+.. post:: July 02, 2024
+  :tags: newsletter
+
+#####################################
+Write the Docs Newsletter – July 2024
+#####################################
+
+Cheers, documentarians! This week has seen a cooling off of temperatures after a heat wave across Europe. Joni Mitchell sang that you don't know what you got till it's gone, and it seems the same is true for things you don't want. I hope you are all able to avoid any unwelcome extremes in your lives, whatever forms they make take.
```

Comment:
```suggestion
Cheers, documentarians! This week has seen a cooling off of temperatures after a heat wave across Europe. Joni Mitchell sang that you don't know what you got till it's gone, and it seems the same is true for things you don't want. I hope you can all avoid any unwelcome extremes in your lives, whatever forms they may take.
```

Reply:
```suggestion
Cheers, documentarians! This week has seen a cooling off of temperatures after a heat wave across Europe. Joni Mitchell sang that you don't know what you got till it's gone, and it seems the same is true for things you don't want. I hope you are all able to avoid any unwelcome extremes in your lives, whatever forms they may take.
```